code/modules/integrated_electronics/subtypes/input.dm

```diff
@@ -130,7 +176,7 @@
 	var/mob/living/carbon/human/H = get_pin_data_as_type(IC_INPUT, 1, /mob/living/carbon/human)
 	if(!istype(H)) //Invalid input
 		return
-	if(H.Adjacent(get_turf(src))) // Like normal analysers, it can't be used at range.
+	if(H in view(get_turf(H))) // Like medbot's analyzer it can be used in range..
```

Comment:
Why was this changed, precisely?

Reply:
This was changed to allow remote work of health sensor. I think, that it should be posible to recreate functions of any bot(farmbot,medbot,securitron, cleanbot,floorbot) with circuitry arsenal. For means of logic and more freedom of creativity.
Medbot can read this params in line of sight.So electrical bot should too.